webpack/envs/clientNovoDevelopmentConfig.js

```diff
@@ -0,0 +1,221 @@
+// @ts-check
+
+const { basePath, env } = require("../utils/env")
+const { getCSSManifest } = require("../utils/getCSSManifest")
+const { HashedModuleIdsPlugin } = require("webpack")
+const { RetryChunkLoadPlugin } = require("webpack-retry-chunk-load-plugin")
+const path = require("path")
+const webpack = require("webpack")
+const WebpackManifestPlugin = require("webpack-manifest-plugin")
+
+export const clientNovoDevelopmentConfig = {
+  stats: "normal",
+  parallelism: 100,
+  mode: env.webpackDebug ? "development" : env.nodeEnv,
+  devtool: "source-map",
+  entry: {
+    "artsy-novo": [path.resolve(process.cwd(), "src/novo/src/client.tsx")],
+  },
```

Comment:
Carrying over unresolved question from previous PR: 

Is there anything different between novo config and other client config besides entry?

Reply:
As stated previously clean separation is preferred for the time being over deduplication.